SPECS/openssh/openssh.spec

```diff
@@ -92,32 +98,32 @@ install -m644 contrib/ssh-copy-id.1 %{buildroot}/%{_mandir}/man1/
 if ! getent passwd sshd >/dev/null; then
    useradd sshd
 fi
-if [ ! -d /var/lib/sshd ]; then
-   mkdir /var/lib/sshd
-   chmod 0755 /var/lib/sshd
+if [ ! -d %{_sharedstatedir}/sshd ]; then
+   mkdir %{_sharedstatedir}/sshd
+   chmod 0755 %{_sharedstatedir}/sshd
 fi
-cp %{buildroot}/usr/bin/scp /usr/bin
+cp %{buildroot}%{_bindir}/scp %{_bindir}
 chmod g+w . -R
 useradd test -G root -m
 sudo -u test -s /bin/bash -c "PATH=$PATH make tests"
 
 %pre server
 getent group sshd >/dev/null || groupadd -g 50 sshd
-getent passwd sshd >/dev/null || useradd -c 'sshd PrivSep' -d /var/lib/sshd -g sshd -s /bin/false -u 50 sshd
+getent passwd sshd >/dev/null || useradd -c 'sshd PrivSep' -d %{_sharedstatedir}/sshd -g sshd -s /bin/false -u 50 sshd
 
 %preun server
 %systemd_preun sshd.service sshd-keygen.service
 
 %post server
 /sbin/ldconfig
 if [ $1 -eq 1 ] ; then
-    chown -v root:sys /var/lib/sshd
+    chown -v root:sys %{_sharedstatedir}/sshd
 fi
-%systemd_post sshd.service sshd-keygen.service
+%{systemd_post} sshd.service sshd-keygen.service
 
 %postun server
 /sbin/ldconfig
-%systemd_postun_with_restart sshd.service sshd-keygen.service
+%{systemd_postun_with_restart} sshd.service sshd-keygen.service
```

Comment:
The linter introduced a bug here- %systemd_postun_with_restart should not have brackets. This is updated in the latest version of the tool.